ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/inventory/host_vars/cv_server.yml

```diff
@@ -1,2 +1,2 @@
 ---
-ansible_python_interpreter: $(which python)
+ansible_python_interpreter: $(which python3)
```

Comment:
I think it's better to have python3 instead of python. Because others could face similar issues. Having python3 won't not create the issues for new developers. Any comment @ClausHolbechArista

Reply:
Yes exactly. I asked Karl to change this on the main conversation :)